app/plugins/download-images.js

```diff
@@ -3,22 +3,56 @@ export const commands = [
   'download-all-images',
 ]
 
-export default async function() {
+export default async function () {
+  if (window.showDirectoryPicker === undefined) {
+    console.log('no directory support :(')
+    return;
+  }
   const dirHandle = await window.showDirectoryPicker()
   const imgs = document.querySelectorAll("img")
-  let i = 0
 
   imgs.forEach(async (img) => {
     const url = img.src
-    const name = `img-${i}.png`
-    i++
-
+    const name = url.substr(url.lastIndexOf('/') + 1);
     try {
-      console.log(`Fetching ${url}`)
       const response = await fetch(url)
```

Comment:
I think we need to do some modifying of image urls too, like we are with css urls. I ran the plugin on css-tricks.com and the script did not properly download everything. It missed over half. Still exciting to see it at work though. Things like ads, that are secure and serve once, I'm totally cool with not downloading. But avatars and other stuff, I def expect those to be in my folder, and they're currently missing for ~40% of css-tricks.com right now. 

I didnt dig into it too deep, but it was urls like this one `https://secure.gravatar.com/avatar/8081b26e05bb4354f7d65ffc34cbbd67?s=80&d=retro&r=pg` that didnt download.

Reply:
okay @argyleink  le me check for it and i'll update you with that tomorrow... now i have some important work to finish off.

Reply:
hey @argyleink I have used the same url and it downloaded the image.
check the screenshots i have attached.
![Screenshot (45)](https://user-images.githubusercontent.com/71861291/103442589-933fff80-4c7d-11eb-8789-c805f0ccad15.png)
![Screenshot (46)](https://user-images.githubusercontent.com/71861291/103442590-963af000-4c7d-11eb-8ec1-b701f3b9bf09.png)

Reply:
i'll look into this further and see what's going on different on css-tricks.com than the visbug test page.